src/infra/monitoring/grafana/terraform/stamps/app-service.tf

```diff
@@ -27,7 +27,7 @@ resource "azurerm_linux_web_app" "appservice" {
   // these env variables are specific to postgres backend supported by grafana
   app_settings = {
     "GF_DATABASE_TYPE"     = "postgres"
-    "GF_DATABASE_HOST"     = "${each.key == "primary" ? azurerm_postgresql_server.pgprimary.name : azurerm_postgresql_server.pgreplica.name}.postgres.database.azure.com"
```

Comment:
@nielsams had to switch to the privatelink dns name to make it work. Wonder why this was working before. Public access to the PGSQL is/was disabled all the time.

The secondary region can probably not reach the primary PGSQL (and vice versa).